javascript/modules/client/README.md

```diff
@@ -441,25 +677,326 @@ console.log(proposals);
+  try {
+    switch (step.key) {
+      case DaoCreationSteps.CREATING:
+        // Transaction waiting on the mempool
+        console.log(step.txHash);
+        break;
+      case DaoCreationSteps.DONE:
+        // DAO address
+        console.log(step.address);
+        break;
+    }
+  } catch (err) {
+    // ...
+  }
+}
+```
 
-### Creating a multisig proposal
+### Creating a Multisig proposal
+```ts
+import {
+  ClientMultisig,
+  DaoAction,
+  ICreateProposal,
+  VoteOption,
+  ContextPlugin,
+  ProposalCreationSteps
+} from "@aragon/sdk-client";
+// context
+const contextPlugin = ContextPlugin.fromContext(context, "0x...")
+const client = new ClientMultisig(contextPlugin);
 
-- **TODO**
+// actions to execute if the proposal is apporved
+const actions: DaoAction[] = [
+  // See the "Action encoders" section below
+  { to: "0x1234...", value: BigInt(100), data: new Uint8Array([1, 2, 3, 4]) },
+];
+// proposal params
+const proposalCreationParams: ICreateProposal = {
+  metadataUri: "<uri>", // Following the EIP-4824
+  actions,
+  startDate: new Date(), // Date
+  endDate: new Date(),  // Date
+  executeOnPass: true,
+  creatorVote: VoteOption.YES,
+};
 
-### Voting on a multisig proposal
+const estimatedGas = await client.estimation.createProposal(
+  proposalCreationParams,
+);
+console.log(estimatedGas.average); // bigint
+console.log(estimatedGas.max); // bigint
+
+for await (
+  const step of client.methods.createProposal(
+    proposalCreationParams,
+  )
+) {
+  try {
+    switch (step.idx) {
+      case ProposalCreationSteps.CREATING:
+        console.log(step.txHash); // 0xb1c14a49...
+        break;
+      case ProposalCreationSteps.DONE:
+        console.log(step.proposalId); // 0x1234...
+        break;
+    }
+  }
+  catch (err){
+    // ...err
+  }
+}
+```
+
+### Voting on a Multisig proposal
+```ts
+import {
+  ClientMultisig,
+  DaoAction,
+  ICreateProposal,
+  VoteOption,
+  ContextPlugin,
+  VoteProposalStep
+} from "@aragon/sdk-client";
+// context
+const contextPlugin = ContextPlugin.fromContext(context, "0x...")
+const client = new ClientMultisig(contextPlugin);
+
+// gas estimation
+const estimatedGas = await client.estimation.voteProposal(
+  proposalId, VoteOptions.YES
+);
+console.log(estimatedGas.average); // bigint
+console.log(estimatedGas.max); // bigint
+
+for await (const step of client.methods.voteProposal(proposalId, VoteOptions.YES)) {
+  try {
+    switch (step.key) {
+    case VoteProposalStep.VOTING:
+        console.log(step.txHash); // 0xb1c14a49...
+      break;
+    case VoteProposalStep.DONE:
+        console.log(step.voteId); // 0x12345...
+      break;
+    }
+  } catch (err) {
+    // ...
+  }
+}
+```
 
-- **TODO**
+### Loading the list of members (Multisig)
+```ts
+import { ClientMultisig, ContextPlugin } from "@aragon/sdk-client";
+// create client
+const contextPlugin = ContextPlugin.fromContext(context, "0x...")
+const client = new ClientMultisig(contextPlugin);
```

Comment:
General renaming suggested: `contextPlugin` => pluginContext (just for the variable)